signature/policy_types.go

```diff
@@ -79,8 +79,18 @@ type prSignedBy struct {
 	// SignedIdentity specifies what image identity the signature must be claiming about the image.
 	// Defaults to "match-exact" if not specified.
 	SignedIdentity PolicyReferenceMatch `json:"signedIdentity"`
+
+	// ReferencesByDigest specifies whether the signature is valid when the reference isn't tagged (e.g. canonical).
+	ReferencesByDigest sbUntaggedReferenceMatch `json:"referencesByDigest"`
```

Comment:
The field name and the type name and the enum values all need to be named consistently.